src/main/main.ts

```diff
@@ -667,27 +663,34 @@ async function initializeStandalone(isRestart: boolean = false): Promise<void> {
       );
     } else if (snapshotPaths.length > 0 && win != null) {
       const snapshotDownloadUrls: string[] = electronStore.get("SnapshotPaths");
-      if (snapshotDownloadUrls.length > 0 && win != null && !isRestart) {
```

Comment:
Is it okay that we ignore `isRestart` here?

Reply:
Yes. The reason `isRestart` was used was because of the old partition snapshot implementation and is no longer needed.